apps/web/pages/[user].tsx

```diff
@@ -284,7 +287,7 @@ export const getServerSideProps = async (context: GetServerSidePropsContext) =>
     avatar: `${WEBAPP_URL}/${user.username}/avatar.png`,
   }));
 
-  if (!users.length) {
+  if (!users.length || (!isValidOrgDomain && !users.some((user) => user.organizationId === null))) {
```

Comment:
We are just ensuring here that in non-org context, org users aren't entertained.
But we still accept user of second org in an org.

I had two users `hariom` in 2 different orgs(org1, org2) and when I went to org1.cal.local/hariom it considered it a dynamic booking link, because it got 2 results.

Reply:
Gotcha, good catch @hariombalhara. I added the org slug when available to narrow the user findings.